compiler/next/include/chpl/uast/TupleDecl.h

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2021 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CHPL_UAST_TUPLEDECL_H
+#define CHPL_UAST_TUPLEDECL_H
+
+#include "chpl/uast/Decl.h"
+#include "chpl/uast/Variable.h"
+#include "chpl/queries/Location.h"
+
+#include <cassert>
+
+namespace chpl {
+namespace uast {
+
+
+/**
+  This class represents a tuple variable declaration
+ 
+  E.g.
+
+  \rst
+  .. code-block:: chapel
+
+      var (a, b) = (1,2);
+      var (c, _) = (3.0,"hi");
+      var (d, e) = returnTuple();
+
+  \endrst
+
+  Each of the lines above is represented by a MultiDecl containing a
+  list of VariableDecls.  Note that the initial value and/or type is inferred
+  from later declarations.
+
+  Since the Tuple does not itself have a name, it is not
+  a NamedDecl. Rather, it can contain NamedDecls.
```

Comment:
Same as for MultiDecls:

I suggest you move the first of these sentences to the previous paragraph and drop the second sentence. The second sentence duplicates what is already written above, with less precision.

Also please extend "MultiDecl containing a list of VariableDecls" to include Tuples and Comments.